ruby-coding-standards.md

```diff
@@ -0,0 +1,23 @@
+# Ruby coding standards
+
+Use the latest version of Ruby 2.4 when starting a new project.
```

Comment:
I'm not really qualified to provide any sort of opinion on Ruby, so this comment may be invalid because of the way Ruby versioning and EOL works, but I'd be interested to know whether we could say something here about what projects that _aren't_ new should do?

I feel like we should have a standard that sets some expectation of not sliding too far behind, which individual applications can be measured against.

Something like:
> New projects should use the latest version of Ruby 2.6
> Existing solutions should not be running on any version of Ruby prior to the latest 2.4